packages/fxa-auth-server/config/index.ts

```diff
@@ -1062,7 +1062,7 @@ const conf = convict({
       accessToken: {
         doc: 'Access Tokens maximum expiration (can live shorter)',
         format: 'duration',
-        default: '2 weeks',
+        default: '365 minutes',
```

Comment:
nit: it may be worth a brief inline comment to explain this weird number, lest our future selves mistake it for a typo :-)

Reply:
@rfk added perhaps excessively verbose comment here 🤷 feedback welcome

Reply:
Haha, you did not muck around with this one, thanks!